src/dodal/devices/pressure_jump_cell.py

```diff
@@ -0,0 +1,239 @@
+from ophyd_async.core import ConfigSignal, StandardReadable
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+
+class PressureJumpCellPumpMode:
+    MANUAL = "Manual"
+    AUTO_PRESSURE = "Auto Pressure"
+    AUTO_POSITION = "Auto Position"
+
+
+class PressureJumpCellBusyStatus:
+    IDLE = "Idle"
+    BUSY = "Busy"
+
+
+class PressureJumpCellTimerState:
+    TIMEOUT = "TIMEOUT"
+    COUNTDOWN = "COUNTDOWN"
+
+
+class PressureJumpCellStopValue:
+    CONTINUE = "CONTINUE"
+    STOP = "STOP"
+
+
+class PressureJumpCellValveControlRequest:
+    OPEN = "Open"
+    CLOSE = "Close"
+    RESET = "Reset"
+    ARM = "Arm"
+    DISARM = "Disarm"
+    # TODO the nones may not be required but FVST and SXST set
+    NONE1 = ""
+    NONE2 = ""
+
+
+class PressureJumpCellPumpMotorControlRequest:
+    ENABLE = "Enable"
+    DISABLE = "Disable"
+    RESET = "Reset"
+    FORWARD = "Forward"
+    REVERSE = "Reverse"
+
+
+class PressureJumpCellPumpMotorDirection:
+    ZERO = "0"
+    FORWARD = "Forward"
+    REVERSE = "Reverse"
+    THREE = "3"
+    FOUR = "4"
+    FIVE = "5"
+    SIX = "6"
+    SEVEN = "7"
+
+
+class PressureJumpCellValveState:
+    FAULT = "Fault"
+    OPEN = "Open"
+    OPENING = "Opening"
+    CLOSED = "Closed"
+    CLOSING = "Closing"
+    NONE5 = ""
+    NONE6 = ""
+
+
+class PressureJumpCellFastValveState:
+    FAULT = "Fault"
+    OPEN = "Open"
+    OPEN_ARMED = "Open Armed"
+    CLOSED = "Closed"
+    CLOSED_ARMED = "Closed Armed"
+    NONE5 = "Unused"
+    NONE6 = ""
+
+
+class PressureJumpCellLimitSwitch:
+    OFF = "Off"
+    ON = "On"
+
+
+class PressureJumpCell(StandardReadable):
+    """
+    High pressure X-ray cell, used to apply pressure or pressure jumps to a sample.
+    """
+
+    def __init__(
+        self,
+        prefix: str = "",
+        adc1_prefix: str = "",
+        adc2_prefix: str = "",
+        name: str = "",
+    ) -> None:
+        with self.add_children_as_readables():
+            ## Valves ##
+            self.valve1_state = epics_signal_r(
+                PressureJumpCellValveState, prefix + "V1:STA"
+            )
+            # V2 - valve manual control
+            self.valve3_state = epics_signal_r(
+                PressureJumpCellValveState, prefix + "V3:STA"
+            )
+            # V4 - valve manual control
+            self.valve5_state = epics_signal_r(
+                PressureJumpCellFastValveState, prefix + "V5:STA"
+            )
+            self.valve6_state = epics_signal_r(
+                PressureJumpCellFastValveState, prefix + "V6:STA"
+            )
```

Comment:
Could: Take a look at `DeviceVector` you can get all these in a list and it will be cleaner I think

Reply:
Thanks I hadn't noticed that, agree it would help reduce the repetition.

Reply:
could: make the open + control as a composite device as they seem to occur together

Reply:
update: no longer necessary